jsonschema-hyperschema.xml

```diff
@@ -388,35 +388,92 @@
+                    are ignored when performing operations to which they are not semantically
+                    relevant.
                 </t>
+                <section title="Resolving templated URIs">
+                    <t>
+                        URI Template variables in <xref target="href">"href"</xref> resolve from
+                        server-supplied instance data by default.
+                        <xref target="hrefSchema">"hrefSchema"</xref> allows a link to specify
+                        a schema for resolving template variables from client-supplied data.
+                        Regular JSON Schema validation features can be used to require resolution
+                        for client data, forbid it, or allow client data while falling back to
+                        server-supplied instance data if no client data is provided.
+                    </t>
+                    <t>
+                        This offers a superset of traditional HTML GET forms, and a subset of all
+                        possible URI construction that can be performed by JavaScript.  To simulate
+                        an HTML GET form, a link can require client input to all query string variables,
+                        and forbid client input for all other template variables.
+                    </t>
+                </section>
+                <section title="Manipulating the target resource representation">
+                    <t>
+                        HTML does not directly support target resource representation manipulation
+                        through forms.  Instead, JavaScript accomplishes this by intercepting the
+                        form submission and using the appropriate HTTP request with the form data.
+                        In JSON Hyper-Schema, <xref target="targetSchema">"targetSchema"</xref>
+                        supplies a non-authoritative description of the target resource's representation.
+                        A client can use "targetSchema" to structure input, or (if it is absent or
+                        if the client prefers to only use authoritative information), interact with
+                        the target resource to confirm or discover its representation structure.
+                    </t>
+                </section>
+                <section title="Submitting data for processing">
+                    <t>
+                        The <xref target="submissionSchema">"submissionSchema"</xref> and
+                        <xref target="submissionEncType">"submissionEncType"</xref> keywords
+                        describe the domain of the processing function implemented by the target resource.
+                        This directly corresponds to HTML POST forms.  The <xref target="form">"form"</xref>
+                        keyword can be used to enforce strict semantic compatibility with HTML POST
```

Comment:
Is "HTML POST form" the right way to reference this?

The use of capital "POST" seems to refer (by convention) to the HTTP method POST.  However, the functionality we are referencing is a form's `method="post"` functionality which is not the same thing as HTTP POST.